x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/custom_time_range_picker.tsx

```diff
@@ -73,8 +77,18 @@ export const CustomTimeRangePicker: FC<CustomUrlTimeRangePickerProps> = ({
 
   return (
     <>
-      <EuiSpacer size="s" />
-      <EuiFlexGroup gutterSize={'none'}>
+      <EuiFlexGroup gutterSize={'none'} alignItems="center">
+        <EuiFlexItem grow={false}>
+          <EuiSwitch
+            disabled={disabled}
+            label={i18n.translate('xpack.ml.customUrlsEditor.addCustomTimeRangeSwitchLabel', {
+              defaultMessage: 'Add custom time range?',
+            })}
+            checked={showCustomTimeRangeSelector}
+            onChange={(e) => onCustomTimeRangeSwitchChange(e.target.checked)}
+            compressed
+          />
+        </EuiFlexItem>
         <EuiFlexItem grow={false}>
           <EuiIconTip
```

Comment:
Can you move the info tooltip to the right of the `Time range` label at the top of this section, as it applies if neither of these switches are enabled?

<img width="350" alt="image" src="https://github.com/elastic/kibana/assets/7405507/6c72b38f-3c14-430b-9589-d3bc992ded02">

Reply:
Moved in 1f35af089e1b012e18376d9ea2654ba34b7fde3b